pricing.js

```diff
@@ -55,22 +55,37 @@ function calculateLTDPrice(product, employee, selectedOptions) {
   return price
 }
 
+function calculateCommuterPrice(product, selectedOptions) {
+  const selectedCost = product.costs.find((cost) => {
+    return cost.type === selectedOptions.benefit
+  })
+  return selectedCost.price
+}
+
```

Comment:
Nice function, that integration test will help you wire it up to `calculateProductPrice`